File: backend/src/main/java/com/bloodline/bloodline_backend/entity/Service.java
package com.bloodline.bloodline_backend.entity;

import jakarta.persistence.*;
import lombok.Data;
import java.math.BigDecimal;

@Data
@Entity
@Table(name = "Service")
public class Service {
    @Id
    @Column(name = "serviceID")
    private Integer serviceID;

    @Column(name = "serviceName", nullable = false)
    private String serviceName;

    @Column(name = "typeOfService", nullable = false)
    private String typeOfService;

    @Column(name = "typeSample", nullable = false)
    private String typeSample;

    @Column(name = "priceSur", nullable = false)
    private BigDecimal priceSur;

    @Column(name = "status", nullable = false)
    private String status;
} 

File: frontend/src/models/Service.js
export class Service {
    constructor(data) {
        this.serviceID = data.serviceID;
        this.serviceName = data.serviceName;
        this.typeOfService = data.typeOfService;
        this.typeSample = data.typeSample;
        this.priceSur = data.priceSur;
        this.status = data.status;
    }
}

export class ServicePrice {
    constructor(data) {
        this.servicePriceID = data.servicePriceID;
        this.serviceID = data.serviceID;
        this.basePrice = data.basePrice;
        this.thirdSamplePrice = data.thirdSamplePrice;
    }
} 